replica/diff_disk.go

```diff
@@ -186,7 +183,9 @@ func (d *diffDisk) fullWriteAt(buf []byte, offset int64) (int, error) {
 				length++
 			}
 		}
+		d.rmLock.Lock()
```

Comment:
`UsedBlocks` seems to be going wrong due to this and previous 'space reclaimation' PR.
Can you explain me the usage of `d.UsedBlocks`?